docs/guide/apm-doc-directory.asciidoc

```diff
@@ -0,0 +1,86 @@
+[[doc-directory]]
+== Documentation directory
+
+The following sections are designed to guide you through the available Elastic APM documentation.
+
+[float]
+=== APM Overview
+
+You're here!
+The APM Overview is your launchpad for getting started with APM.
+We recommend reading all sections in this book to gain a deeper understanding of the Elastic APM ecosystem.
+
+* <<overview>>
+* <<install-and-run>>
+* <<apm-data-model>>
+* <<distributed-tracing>>
+* <<agent-server-compatibility>>
+
+TIP: Want to skip straight to installation?
+Simply use an existing cluster or grab a fresh installation of the Elastic Stack,
+spin up an APM Server, and add a bit of code to instrument your app with agents.
+Full details are available on the <<install-and-run,install and run>> page.
+
+[float]
+=== APM Server
+
+The APM Server receives data from APM agents and converts that data into Elasticsearch documents.
+The {apm-server-ref-v}/index.html[APM Server reference] provides everything you need when it comes to working with the server.
+Here you can learn about {apm-server-ref-v}/installing.html[installation],
+{apm-server-ref-v}/configuring-howto-apm-server.html[configuration],
+{apm-server-ref-v}/securing-apm-server.html[security],
+{apm-server-ref-v}/monitoring.html[monitoring], and more.
+
+[float]
+=== APM Agents
+
+APM agents are open source libraries that you install into your services.
+You may only need one, or you might use all six.
```

Comment:
this `all six` might turn obsolete if we add more agents and you miss/forget to update this wording...